staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -339,6 +369,25 @@ func (w *watchCache) WaitUntilFreshAndGet(resourceVersion uint64, key string, tr
 	return value, exists, w.resourceVersion, err
 }
 
+// WaitUntilFreshAndListWithIndex returns a pointers to <storeElement> object, try to list through the index.
+func (w *watchCache) WaitUntilFreshAndListWithIndex(resourceVersion uint64, matchValues []storage.MatchValue, trace *utiltrace.Trace) ([]interface{}, uint64, error) {
```

Comment:
WaitUntilFreshAndList is no longer used - can we simply replace that function with this one?

Reply:
agree it